sandbox/simulator/main

```diff
@@ -113,6 +124,7 @@ $events = db_QueryFetch(
 	);
 
 $eventindex = count($events) + 1;
+$eventsId = $events_root[0]['id'];
```

Comment:
Modified date is unused, you use a LIMIT 1 in the query, and you only care about a single result value. In this case, you can use `db_QueryFetchValue`, and store the returned value in `$eventsId` directly.

Reply:
fixed